packages/devtools_app/lib/src/screens/memory/primitives/simple_elements.dart

```diff
@@ -17,3 +17,49 @@ const String nonGcableInstancesColumnTooltip =
     'Number of instances of the class,\n'
     'that are reachable, i.e. have a retaining path from the root\n'
     'and therefore can’t be garbage collected.';
+
+/// Analytic time constants specific for memory screen.
+class MemoryTimeAnalytics {
```

Comment:
please move both of these constants to analytics_constants.dart. This is where all constants for our analytics live, so let's keep them all in the same place.